codalab/bundles/run_bundle.py

```diff
@@ -29,7 +29,7 @@ class RunBundle(NamedBundle):
     METADATA_SPECS = list(NamedBundle.METADATA_SPECS)
     # Note that these are strings, which need to be parsed
     # Request a machine with this much resources and don't let run exceed these resources
-    METADATA_SPECS.append(MetadataSpec('request_docker_image', basestring, 'Which docker image (e.g., codalab/ubuntu:1.9) we wish to use.'))
+    METADATA_SPECS.append(MetadataSpec('request_docker_image', basestring, 'Which docker image (e.g., codalab/ubuntu:1.9) we wish to use.', choices=("codalab/ubuntu", "codalab/tensorflow-cuda7.0-352.39")))
```

Comment:
'choices' somehow suggests that this is a closed set of options, but it's not in this case - people can request any docker image they want...is this the place we want to add it?

I thought we would want a DockerImageCompleter, with hard-coding for now, but later something that calls 'docker search'.